UnitTests/PayPalTests/PayPalEnvironment_Tests.swift

```diff
@@ -0,0 +1,14 @@
+import XCTest
+@testable import PaymentsCore
+@testable import PayPal
+
+final class PayPalEnvironment_Tests: XCTestCase {
```

Comment:
Do we need `final`? I think we're probably fine without it for our test suite. But if we _really did_ want to add it, we should make it consistent across all test files.

Reply:
yeah +1 on removing it - `final` is only helpful when you never want anyone to be able to subclass your type.